axelor-sale/src/main/resources/views/SaleOrder.xml

```diff
@@ -1520,6 +1550,34 @@
     <action name="action-sale-order-method-finalize-quotation"/>
   </action-group>
 
+  <action-group name="action-sale-order-group-new-version-btn">
+    <action name="save"/>
+    <action name="action-sale-order-validate-check-empty-sale-order-on-new-version"/>
+    <action name="action-sale-order-method-new-version"/>
+    <action name="action-sale-order-attrs-set-previous-version-number"/>
+    <action name="action-sale-order-method-get-version-date-time"/>
+    <action name="save"/>
```

Comment:
To avoid having too many actions, please refactor this action group into only

```xml
    <action name="save"/>
    <action name="action-sale-order-method-new-version-btn"/>
```

by moving the existing actions inside an action method.